.github/workflows/codeql-stats.yml

```
@@ -0,0 +1,28 @@
+name: CodeQL statistics
+
+on:
+  schedule:
+    - cron: '27 0 * * 1' # At 00:27 every Monday
+
+jobs:
+  stats:
+    name: CodeQL statistics
+    runs-on: ubuntu-latest
+    if: github.repository == 'elastic/kibana' # Hack: Do not run on forks
```

If you want to be explicit, this worked for stopping the LaunchDarkly action from running on forks.

```suggestion
    if: github.event.repository.fork == false
```

---

I'm going to hold off for the moment, this is working and I expect it won't cover the "not fork but clone and pushed" edge case.  Not that I expect that to be common, will circle back of there's issues.